src/Sdk/WebApi/WebApi/RawHttpClientBase.cs

```diff
@@ -101,15 +101,55 @@ public void Dispose()
             }
         }
 
+        protected async Task<RawHttpClientResult> Send2Async(
```

Comment:
Should we better name `Send2Async`? This is RawHttp, so not every call needs to have json response like run-service does, in that sense, we might want to keep `SendAsync` and this new one as well?

Reply:
or since we have `try*` anyway, why do we need `send2`? we plan to delete `send` one later? 🤔

Reply:
Open to changing this. I played around with a few different names but everything was horrible.

E.g. "SendAndReceiveWrapperResult". Such a long, aesthetically displeasing name.